CONTRIBUTING.md

```diff
@@ -15,13 +15,14 @@ Thank you for your interest in contributing to Eventbrite's Javascript SDK!
 
 ### Setup
 
-1. Fork the repository
-2. Clone your new forked repository to your local computer
-3. Set the Eventbrite repository as your branches upstream branch
+1. This project uses [yarn](https://yarnpkg.com/en/) as a package management system. If you don't have it installed, you can follow the instructions [here](https://yarnpkg.com/lang/en/docs/install/)
+1. Fork the repository. [need help?](https://help.github.com/articles/fork-a-repo/)
+1. Clone your new forked repository to your local computer
+1. Set the Eventbrite repository as your branches upstream branch
 `git remote add upstream https://github.com/eventbrite/eventbrite-sdk-javascript.git`
-4. Navigate to the root directory of your newly cloned repository 
+1. Navigate to the root directory of your newly cloned repository 
 (for mac `cd /path/to/eventbrite-sdk-javascript`)
-5. `yarn install` to install local dependencies
+1. `yarn install` to install local dependencies
```

Comment:
TIL you can number them all 1 and they will autoincrement.

Reply:
😅 i spent more time working out what @benmvp was trying to say here than i care to admit

Reply:
Yeah that is super odd, but really cool. I would not have guessed that would have worked.